### PR TITLE
Play onelive homepage youtube videos on the archive page

### LINKED ISSUE #672: Play onelive homepage youtube videos on the page
**Is your feature request related to a problem? Please describe.**
- Currently, there are three main videos on onelive homepage and they play on youtube.
we need to play those videos on the same page.

**Describe the solution you'd like**
- Write a function to play videos on the same page.

**Additional context**
- Please refer to this pr
https://github.com/sef-global/sef-site/pull/668

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #672 

## Goals

- Play onelive homepage youtube videos on the archive page

## Approach

- Play onelive homepage youtube videos on the page

### Preview Link

https://pr-680-sef-site.surge.sh/onelive/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] My code follows the style guidelines of this project
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/onelive/index.html [hunk 1]
```diff
@@ -309,31 +309,36 @@ <h5 class="title">
...
+                 class="img-fluid rounded shadow img-full">
+            <div class="card-body">
+                <h6 class="font-weight-bold">{{snippet.title}}</h6>
+                <h6 class="small">{{snippet.publishedAt}}</h6>
+                <p class="text-muted">{{snippet.description}}</p>
+            </div>
+            <!-- Change button style by changing btn-youtube into anything you like ex: btn-onelive -->
+            <a class="btn btn-onelive position-absolute hover-element rounded-05-rem"
+               style="right: 20px; bottom: 20px"
+               onclick="insertParam('{{snippet.resourceId.videoId}}')">
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html]: just render it with the template.

`<a href="/onelive/archive?videoId={{snippet.resourceId.videoId}}"`

File: src/main/webapp/onelive/archive/functions.js [hunk 2]
```diff
@@ -36,3 +36,9 @@ function loadYoutubeVideoPlayer(videoId) {
     let iframeTag = '<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/' + videoId + '"></iframe>';
     $('#iframe').html(iframeTag);
 }
+
+function loadHomePageVideo() {
+    let videoId = window.location.href.substr(1).split('videoId=')[1];
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/functions.js]: 1st Check if there's a param exists. (there's no videoId param if someone just visit the archive page)

use this way: https://stackoverflow.com/questions/7731778/get-query-string-parameters-url-values-with-jquery-javascript-querystring

File: src/main/webapp/onelive/archive/functions.js [hunk 3]
```diff
@@ -36,3 +36,9 @@ function loadYoutubeVideoPlayer(videoId) {
...
+    console.log(videoId);
+    loadYoutubeVideoPlayer(videoId)
+}
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/functions.js]: EOF

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Nice job @anjula-sack !

Could you please look into the requested changes?

File: src/main/webapp/onelive/index.html [hunk 4]
```diff
@@ -309,30 +309,26 @@ <h5 class="title">
...
-                                <i class="fas fa-play"></i>
-                            </span>
-                    <span class="content-on-hover">
-                                Watch Now &nbsp;<i class="fas fa-play fa-sm"></i>
-                            </span>
-                </a>
+        <div class="card video-card shadow youtube-videos mb-4">
+            <img src="{{snippet.thumbnails.medium.url}}"
+                 alt="Rounded image"
+                 class="img-fluid rounded shadow img-full">
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/index.html]: self-closing tag.  `/>`

Comment (anjula-sack) [reply src/main/webapp/onelive/index.html]: ok

File: src/main/webapp/onelive/archive/functions.js (hunk 3 above)
Comment (anjula-sack) [reply src/main/webapp/onelive/archive/functions.js]: ok

File: src/main/webapp/onelive/archive/functions.js (hunk 2 above)
Comment (anjula-sack) [reply src/main/webapp/onelive/archive/functions.js]: I'll fix it.

File: src/main/webapp/onelive/index.html (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/onelive/index.html]: ok

**Review by jayasanka-sack — APPROVED** [review]
Perfect!

**Review by Piumal1999 — APPROVED** [review]
Awesome!! @anjula-sack

**Pull request merged**